ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/hosts_file/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+#####################
+# Update /etc/hosts #
+#####################
+---
+- name: Update /etc/hosts file - IP FQDN hostname
+  lineinfile:
+    dest: /etc/hosts
+    regexp: "^(.*){{ ansible_hostname }}(.*)$"
+    line: "{{ ansible_default_ipv4.address }} {{ inventory_hostname }}.{{ Domain }} {{ inventory_hostname }}"
+    state: present
```

Comment:
* Note #1996 and use new identifier 
* Also, much prefer the shortname (ie, no domain) as first argument and hostname with domain as a comment (or at least last on line)